### PR TITLE
Support to CPF and CNPJ

### PR DESCRIPTION
**Pull request by CristianDRM**
- Refactory to conform with SOLID Pattern
 - Add support to CPF and CPNJ mask

### CODE REVIEW

File: GeneralFormatter.podspec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'GeneralFormatter'
-  s.version          = '1.1.1'
+  s.version          = '2'
```
Comment (Morbix) [inline GeneralFormatter.podspec]: Prefer `2.0.0`

File: Example/Pods/Target Support Files/GeneralFormatter/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
   <key>CFBundlePackageType</key>
   <string>FMWK</string>
   <key>CFBundleShortVersionString</key>
-  <string>1.0.0</string>
+  <string>1.1.1</string>
```
Comment (Morbix) [inline Example/Pods/Target Support Files/GeneralFormatter/Info.plist]: Why it is 1.1.1?

File: GeneralFormatter/Classes/CNPJFormatter.swift [hunk 3]
```diff
@@ -0,0 +1,37 @@
+//
+//  CNPJFormatter.swift
+//  Pods
+//
+//  Created by Cristian Madrid on 22/06/17.
+//
+//
```
Comment (Morbix) [inline GeneralFormatter/Classes/CNPJFormatter.swift]: Remove all headers

File: Example/GeneralFormatter/Base.lproj/Main.storyboard [hunk 4]
```diff
@@ -21,26 +21,101 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
```
Comment (Morbix) [inline Example/GeneralFormatter/Base.lproj/Main.storyboard:21]: Update README

**Review by Morbix — CHANGES_REQUESTED** [review]
Fix comments

**Comment by Morbix** [comment]
By the way, very nice Refactor.

**Comment by Morbix** [comment]
@UrielBattanoli I'll need release a new version of the pod, could you add me as a contributor, please? 

You need register a session first using the command bellow:
```
$ pod trunk register add-your-email-here 'Uriel Battanoli' --description='macbook'
```

You will receive an e-mail, open and click in the link to confirm the registration.

And add me using this command:
```
$ pod trunk add-owner GeneralFormatter morbin_@hotmail.com
```

_Source: https://guides.cocoapods.org/making/getting-setup-with-trunk.html_

**Comment by CristianDRM** [comment]
Reviewed

File: GeneralFormatter/Classes/GeneralFormatter.swift [hunk 5]
```diff
@@ -15,6 +7,7 @@ public enum GeneralFormatter {
     case cpfCnpj
     case phone
     case cep
+    case custom(formatter: Formatter)
```
Comment (Morbix) [inline GeneralFormatter/Classes/GeneralFormatter.swift]: You did again!!! >/

**Review by Morbix — APPROVED** [review]

**Pull request merged**